vmdk_plugin/Makefile

```diff
@@ -95,10 +98,18 @@ VMDKOPS_TEST_MODULE := vmdkops
 VMDKOPS_MODULE_SRC = $(VMDKOPS_MODULE)/*.go $(VMCI_SRC)
 
 # All sources. We rebuild if anything changes here
-SRC = main.go log_formatter.go utils/refcount/refcnt.go \
+SRC = original_plugin/main.go original_plugin/constants.go \
+	utils/refcount/refcnt.go utils/log_formatter/log_formatter.go \
+	utils/plugin_server/plugin_server.go \
 	utils/fs/fs.go utils/config/config.go utils/plugin_utils/plugin_utils.go\
 	drivers/photon/photon_driver.go drivers/vmdk/vmdk_driver.go
 
+SHARED_PLUGIN_SRC = shared_plugin/main.go shared_plugin/constants.go \
```

Comment:
can we use $(subst) to build SHARED_PLUGIN_SRC instead of copy and paste ? https://www.gnu.org/software/make/manual/html_node/Text-Functions.html